webapp/portlet/src/main/webapp/vue-apps/spaces-list/components/ExoSpaceFavoriteAction.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <favorite-button
+    :id="spaceId"
+    type="space"
+    :space-id="spaceId"
+    :favorite="isFavorite"
+    :absolute="absolute"
+    :top="top"
+    :right="right"
```

Comment:
```suggestion
    :space-id="spaceId"
    :favorite="isFavorite"
    :absolute="absolute"
    :top="top"
    :right="right"
    type="space"
```
The constant values should goes after the variabilized ones